pyproject.toml

```diff
@@ -0,0 +1,6 @@
+[build-system]
```

Comment:
Well, learning about PIP518 was a disappointment. Ah well, nothing we can do about that now, I should have followed the PEP discussions in 2016. My bad.

Reply:
I use tox routinely to run linting checks (which admittedly you have none) and then the test suite on several Python versions. I find that this saves me a lot of time compared to just waiting for the CI to run which is not that fast. But it's your project and your call, so do I remove tox.ini?

Reply:
No, we can keep it. I'll think about the src/ change, but we can do it separately.

"The reason is that running the tests in the project directory will import the project source rather than the installed version of the package."

Mmm, yeah, see that's what I like about it. :-)

"(like missing files in MANIFEST.in)."

Ah, I've always used check-manifest for that.